drivers/ethernet/eth_sam_gmac_priv.h

```diff
@@ -194,7 +194,8 @@ enum queue_idx {
 	GMAC_QUE_5,  /** Priority queue 5 */
 };
 
-#if (DT_INST_PROP(0, max_frame_size) == 1518)
+#if (DT_INST_HAS_PROP(0, max_frame_size)) || \
```

Comment:
This changes the logic of these checks. Now if `DT_INST_HAS_PROP(0, max_frame_size)` is true, then the max_frame_size value is not checked at all and the `GMAC_MAX_FRAME_SIZE` is set to 0 always. This does not look a correct thing to do here.

Reply:
ah, good catch.  Will fix, thanks.

Reply:
The **max_frame_size** must be a required = true on my opinion. Otherwise the purpose of the removal of "default" field has no meaning. This property is mandatory and should be adjusted accord with IP Stack. The "default" is always the minimal ethernet frame. With VLAN enabled, frame should be 1536. User can decide not save RAM and enable Jumbo 10240.